### DISCUSSION TITLE
using "default" as x-delayed-type throws an operation exception

### DISCUSSION THREAD

**Discussion by martywaz** [rabbitmq/rabbitmq-delayed-message-exchange#244] [General]
Asking if support for "default" exchange will be added. This would be very useful since if an error is thrown on one subscribing queue, you would not want it redelivered on every subscribing queue...just the queue that generated the exception.

**Comment by michaelklishin** [comment]
There is no such exchange type. The default exchange is not a type, it's a convention (these days it's a special code path).

You can use the built-in direct exchange type named `amq.direct`. The only difference will be that queues must be bound to it explicitly (e.g. with the routing key = to queue name), whereas the default exchange is automatically considered to be bound to every queue in the same virtual host.
